esp/src/src/nls/fr/hpcc.ts

```diff
@@ -125,8 +125,10 @@ export = {
     ContinueWorking: "Continuer à travailler",
     Copied: "Copié",
     Copy: "Copier",
+    CopyLogicalFiles: "Copier Les Fichiers Logiques Au Presse-papier ",
```

Comment:
Capitalization does not match other phrases around here

Reply:
I'll make the correction. good catch

Reply:
I have double check with the strings for english and fixed the ones that do not need to be capitalized.

Reply:
Still look to be a few capitalization inconsistencies...